### DISCUSSION TITLE
Want a javascript library for DaisyUI chat bubbles and toasts

### DISCUSSION THREAD

**Discussion by keanesnow98** [saadeghi/daisyui#2071] [Ideas / feature requests]
I am really satisfied with new DaisyUI 3 components.
I found almost of them are all big help for my projects.
But, some components like chat bubbles and toasts, I also found amazing in them, but I couldn't flexibly use in my project.
I think if there is a JavaScript library for dynamic showing of chat bubbles and toasts, then it would expand DaisyUI users even more!

**Comment by saadeghi** [comment]
What do you want the JS to do? Adding and removing them?

**Reply by keanesnow98** [reply]
Yes, I want dynamic adding and removing.
Is it possible to do that?

**Reply by saadeghi** [reply]
Of course it's possible but it really depends on how you want to use them. And also syntax is different on each JS framework.  

Here's an example of adding removing items with Svelte:  
https://svelte.dev/repl/def193153e304d5f9df6c46463b9fba9?version=4.0.1  

But you can do the same with other JS frameworks, or with pure JS or with [Alpine.js](https://alpinejs.dev), etc.

**Reply by keanesnow98** [reply]
Well, so there aren't any samples exist yet..
Anyway, thanks, @saadeghi !
